### ISSUE TITLE
Supports a table of contents

### ISSUE THREAD

**Issue by sculove** [rhiokim/flybook#20]
you can create a table of  contents using headline title like this

### file A.md
```
## title
### title1
### title2
```

### a table of contents
A
 - title
    - title1
    - title2

How about supporting a table of contents?

**Comment by djkeh** [comment]
I agree that TOC would be a great component for writing markdown document. There might be some points to consider before putting it into flybook as a feature.

* There are some independent TOC builders already in form of package, plugin or extension.
* TOC itself doesn't seem like a markdown standard kinda thing.
* GFM - the most popular markdown language among developers - doesn't support TOC component as a markdown language.
* There's no standardized TOC implementation(design), which means if flybook implements the way to create TOC block it will be flybook-original design.

All concerns above are not the critical reasons flybook shouldn't have TOC feature, but just throwing points to consider whether we go or not.
